bin/run_web.sh

```diff
@@ -21,9 +21,10 @@ gunicorn \
     --reload \
     --bind=0.0.0.0:8000 \
     --workers=1 \
-    --worker-connections=1 \
+    --worker-connections=2 \
```

Comment:
This number now covers incoming connections AND the heartbeat because they're in the same pool. Thus if it's 1, then Antenna hilariously doesn't work at all because the heartbeat takes the slot and keeps it forever.